charts/vela-core/templates/defwithtemplate/affinity.yaml

```diff
@@ -31,6 +31,9 @@ spec:
         						if k.namespace != _|_ {
         							namespace: k.namespace
         						}
+        						if k.namespaces != _|_ {
```

Comment:
Hi, you need to modify the cue files and use make reviewable to generate the yaml changes.

Reply:
see https://github.com/kubevela/kubevela/pull/6556